test/unit/sessions/collection.test.js

```diff
@@ -59,9 +59,9 @@ describe('Sessions', function() {
         const options = Object.freeze({});
         test.server.setMessageHandler(request => {
           const doc = request.document;
-          if (doc.ismaster) {
+          if (doc.ismaster || doc.hello) {
             request.reply(mock.DEFAULT_ISMASTER_36);
-          } else if (doc.count || doc.endSessions) {
+          } else if (doc.count || doc.aggregate || doc.endSessions) {
```

Comment:
We could callout here `// count changed behavior to use agg 3.6.x+`, just a nit, thinking about our future selves.